tests/django_restframework_gis_tests/test_filters.py

```diff
@@ -1,13 +1,15 @@
 import json
 import urllib
 
+from unittest import skipIf
+
+from django.conf import settings
 from django.test import TestCase
 from django.contrib.gis.geos import GEOSGeometry, Polygon
 from django.core.urlresolvers import reverse
 
 from .models import Location
 
-
```

Comment:
could you avoid removing this line please? It's not necessary and we do want to blank lines between class definitions and imports (PEP8).